### PR TITLE
New profile hook and capacities

### PR DESCRIPTION
**Pull request by flexsurfer**
Hooks:
`profile.settings`

Events:
`store/put-in`

Queries:
`store/get-in      `
`store/get-vals   `  
`time/now   `

_Note to testers_ Regular sanity checks

### CODE REVIEW

File: src/status_im/extensions/core.cljs [hunk 1]
```diff
@@ -322,12 +336,17 @@
...
+                     auto-focus on-submit default-value]}]
   [react/text-input (merge {:placeholder placeholder}
                            (when placeholder-text-color {:placeholder-text-color placeholder-text-color})
                            (when selection-color {:selection-color selection-color})
                            (when style {:style style})
                            (when keyboard-type {:keyboard-type keyboard-type})
+                           (when auto-focus {:auto-focus auto-focus})
+                           (when default-value {:default-value default-value})
+                           (when on-submit
+                             {:on-submit-editing #(on-submit {})})
```
Comment (jeluard) [inline src/status_im/extensions/core.cljs:349]: Maybe add current value as value?

File: src/status_im/ui/screens/profile/user/views.cljs [hunk 2]
```diff
@@ -274,15 +285,24 @@
      :accessibility-label :notifications-button
      :new?                (not seen?)
      :action-fn           #(re-frame/dispatch
-                            [:tribute-to-talk.ui/menu-item-pressed])}
+                            [:tribute-to-talk.ui/menu-item-pressed])
```
Comment (jeluard) [inline src/status_im/ui/screens/profile/user/views.cljs]: Intentional?

File: src/status_im/ui/screens/profile/user/views.cljs [hunk 3]
```diff
@@ -324,6 +344,6 @@
          [contacts-list-item active-contacts-count]
          (when config/tr-to-talk-enabled?
            [tribute-to-talk-item snt-amount tribute-to-talk-seen?])
-         [my-profile-settings current-account shown-account currency (nil? login-data)]
+         [my-profile-settings current-account shown-account currency (nil? login-data) extensions]
```
Comment (jeluard) [inline src/status_im/ui/screens/profile/user/views.cljs:346]: Can we have multiples?

File: src/status_im/extensions/core.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/extensions/core.cljs:349]: yes makes sense

File: src/status_im/ui/screens/profile/user/views.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/profile/user/views.cljs]: no, thanks

File: src/status_im/ui/screens/profile/user/views.cljs (hunk 3 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/profile/user/views.cljs:346]: yes

**Review by jeluard — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

**Comment by asemiankevich** [comment]
@antdanchenko ^ test is failing however works manually. 
сс @Serhy @churik

**Comment by asemiankevich** [comment]
@jakubgs there are always random fails in building artifacts here, please take a look.

**Pull request merged**